### ISSUE TITLE
[Bug]: No alerts set for Windowsclients using windows exporter.

### ISSUE THREAD

**Issue by susfs** [netdata/netdata#15369]
### Bug description

Hi there, 

I am experiencing following issue:
when adding (it does not matter if a virtual node or directly) a windows host, the metrics are recognised but no alarms were automatically set (see screenshot).
If I add a regular linux host, alarms are set.  
![image](https://github.com/netdata/netdata/assets/139250504/03dbaa2b-9b53-4c8d-9bdd-3e4845d360e4)

### Expected behavior

Templated alarms are set automatically.

### Steps to reproduce

1. Install windows exporter on windows host.
2. Add windows host in vhost.conf with newly generated uuid.
3. Add windows host on wmi.conf
4. restart Netdata

### Installation method

docker

### System info

```shell
/etc/alpine-release:3.18.0
/etc/os-release:NAME="Alpine Linux"
/etc/os-release:ID=alpine
/etc/os-release:VERSION_ID=3.18.0
/etc/os-release:PRETTY_NAME="Alpine Linux v3.18"
```

### Netdata build info

```shell
Version: netdata v1.40.0
Configure options:  '--prefix=/usr' '--sysconfdir=/etc' '--localstatedir=/var' '--libexecdir=/usr/libexec' '--libdir=/usr/lib' '--with-zlib' '--with-math' '--with-user=netdata' '--without-bundled-protobuf' '--disable-dependency-tracking' '--disable-ebpf' 'CFLAGS=-O2 -pipe' 'LDFLAGS='
Install type: oci
    Binary architecture: x86_64
Features:
    dbengine:                   YES
    Native HTTPS:               YES
    Netdata Cloud:              YES
    ACLK:                       YES
    TLS Host Verification:      YES
    Machine Learning:           YES
    Stream Compression:         YES
    HTTPD (h2o):                YES
Libraries:
    protobuf:                YES (system)
    jemalloc:                NO
    JSON-C:                  YES
    libcap:                  NO
    libcrypto:               YES
    libm:                    YES
    tcalloc:                 NO
    zlib:                    YES
Plugins:
    apps:                    YES
    cgroup Network Tracking: YES
    CUPS:                    NO
    debugfs:                 YES
    EBPF:                    NO
    IPMI:                    YES
    NFACCT:                  NO
    perf:                    YES
    slabinfo:                YES
    Xen:                     NO
    Xen VBD Error Tracking:  NO
Exporters:
    AWS Kinesis:             NO
    GCP PubSub:              NO
    MongoDB:                 YES
    Prometheus Remote Write: YES
Debug/Developer Features:
    Trace Allocations:       NO
```

**Comment by ilyam8** [comment]
Hey, @susfs. Thanks for reporting. Fixed in #15376.

**Comment by susfs** [comment]
Hi @ilyam8, 

it worked! Thank you very much for the fast response and fix!
